cloudslang-cli/src/main/resources/configuration/cslang.properties

```diff
@@ -7,4 +7,5 @@ maven.settings.xml.path=${app.home}/maven/conf/settings.xml
 maven.m2.conf.path=${app.home}/maven/conf/m2.conf
 cloudslang.maven.repo.local=${app.home}/maven/repo
 cloudslang.maven.repo.remote.url=http://repo1.maven.org/maven2
-cloudslang.maven.plugins.remote.url=http://repo1.maven.org/maven2
\ No newline at end of file
+cloudslang.maven.plugins.remote.url=http://repo1.maven.org/maven2
```

Comment:
since there is no change in this file , plesae rever to original and do not check-in

Reply:
we are pointing to the external python here